kythe/go/extractors/config/extractor.go

```diff
@@ -26,10 +26,13 @@ import (
 	"os/exec"
 	"path/filepath"
 	"strings"
+
+	ecpb "kythe.io/kythe/proto/extraction_config_go_proto"
 )
 
 // kytheConfigFileName The name of the Kythe extraction config
 const kytheExtractionConfigFile = ".kythe-extraction-config"
+const defaultConfigDir = "kythe/go/extractors/config/default"
```

Comment:
It seems like this must necessarily be a stopgap. Can you please add a TODO(#xyz) to point to the appropriate issue?

Reply:
yea ok we can ax this.  The relevant TODO down below covers it (make configurable)